src/main/java/org/jmxtrans/embedded/EmbeddedJmxTrans.java

```diff
@@ -522,4 +526,14 @@ public int getDiscardedResultsCount() {
         }
         return result;
     }
+
+    @Nullable
+    public String getState() {
```

Comment:
Since the state enum is package private might make sense having this package private too since the states are not available and returning the enum without tostring.